index.html

```diff
@@ -0,0 +1,176 @@
+    <aside class="grid1">
+      <article>
+       <img src="img/Snapshoot(2).png" alt="Snapshoot Portfolio">
+        <div class="contents">
+         <h2>Tonic</h2>
+         <ul>
+           <li class="brand">CANOPY</li>
+           <li><img src="img/Counter.png" alt="counter"> Back End Dev</li>
+           <li><img src="img/Counter.png" alt="counter"> 2015</li>
+         </ul>
+         <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+         <ul class="li2">
+          <li>html</li>
+          <li>css</li>
+          <li>javaScript</li>
+         </ul>
+         <button type="submit">See Project</button>
+       </div>
+    </article>
+   </aside>  
+ </div>
+<!--Next page -->
+  <div class="continer">
+    <aside class="grid1">
+      <article>
+      <img src="img/Snapshoot(3).png" alt="Snapshoot Portfolio">
+    <div class="contents">
+      <h2>Multi-Post<br>Stories</h2>
+      <ul>
+        <li class="brand">CANOPY</li>
+        <li><img src="img/Counter.png" alt="counter"> Back End Dev</li>
+        <li><img src="img/Counter.png" alt="counter"> 2015</li>
+      </ul>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <ul class="li2">
+        <li>html</li>
+        <li>css</li>
+        <li>javaScript</li>
+      </ul>
+      <button type="submit">See Project</button>
+    </div>
+    </article>
+    </aside> 
+  </div>
+<!--Next page --> 
+    <section>
+      <h1>About<br>Myself</h1>
+      <p class="p1">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      <p class="p2">LET’S CONNECT</p>
+    <ul class="images">
+      <li><a href=""><img src="img/Vector.png" alt="icon1"></a></li>
+      <li><a href=""><img src="img/Linkedin.png" alt="icon2"></a></li>
+      <li><a href=""><img src="img/Vector(1).png" alt="icon3"></a></li>
+      <li><a href=""><img src="img/Vector(2).png" alt="icon4"></a></li>
+      <li><a href=""><img src="img/Vector(3).png" alt="icon5"></a></li>
+    </ul>
+  </section>
+</section>
+  <aside>
+  <div class="lang">
+    <h2>Languages</h2>
+    <img src="img/Icon2.png" alt="icon">
+  </div>
+  <div class="skills">
+    <div>
+      <img src="img/Ellipse.png" alt="icon">
+      <p>JavaScript</p>
+    </div>
+  </div>
+  <div class="skills">
+    <div>
+      <img src="img/Ellipse(1).png" alt="icon5">
+      <p class="margin">HTML</p>
+    </div>
+  </div><div class="skills">
+    <div>
+      <img src="img/Ellipse(2).png" alt="icon4">
+      <p class="margin">CSS</p>
+    </div>
+  </div>
+  <div class="lang1">
+    <h2>Frameworks</h2>
+    <img src="img/Icon1.png" alt="icon">
+  </div>
+  <div class="line"></div>
+  <div class="lang2">
+    <h2>Skills</h2>
+    <img src="img/Icon1.png" alt="icon">
+  </div>
+  <div class="line"></div>
+  </aside>
```

Comment:
- [ ] **[OPTIONAL]**

- Although the previous reviewer didn't point this out, do bear in mind that they're **MANDATORY REQUIREMENT** for this project...
- Kindly use `ul` for your skills.